Author: <doko@salsa.net.local>
Description: Don't set a runtime path for linking the shared libraries
diff -urNad readline5-5.2~/support/shobj-conf readline5-5.2/support/shobj-conf
--- readline5-5.2~/support/shobj-conf	2006-04-11 15:15:43.000000000 +0200
+++ readline5-5.2/support/shobj-conf	2006-10-28 14:21:36.000000000 +0200
@@ -110,7 +110,7 @@
 	SHOBJ_LD='${CC}'
 	SHOBJ_LDFLAGS='-shared -Wl,-soname,$@'
 
-	SHLIB_XLDFLAGS='-Wl,-rpath,$(libdir) -Wl,-soname,`basename $@ $(SHLIB_MINOR)`'
+	SHLIB_XLDFLAGS='-Wl,-soname,`basename $@ $(SHLIB_MINOR)`'
 	SHLIB_LIBVERSION='$(SHLIB_LIBSUFF).$(SHLIB_MAJOR)$(SHLIB_MINOR)'
 	;;
 
